fairlearn/reductions/_moments/utility_parity.py

```diff
@@ -282,12 +374,30 @@ class TruePositiveRateParity(UtilityParity):
 
     short_name = "TruePositiveRateParity"
 
-    def load_data(self, X, y, **kwargs):
```

Comment:
We should be able to change this signature as well as other moments derived from `UtilityRate` to something like:
```python
    def load_data(self, X, y, *, sensitive_features, control_features=None):
```

Reply:
Done